### ISSUE TITLE
Further suggested enhancements

### ISSUE THREAD

**Issue by stewrg** [tridentli/trident#57]
Ability to tick a 'send update notification' email to group when an update is made to the WIKI or a new file is uploaded to Files.... including a hyperlink to the relevant Wiki page or file.

**Comment by bapril** [comment]
Would it make more or less sense to do this in the form of 2 dedicated mailing-lists? e.g. $tg-wiki and $tg-file where members can subscribe or not and get a stream of all updates?

**Comment by stewrg** [comment]
Yes - that would work well - with the default set to "notify Group Wiki/File Updates" set to "Yes"
